lib/wise_homex/models/ds_energy_meter.ex

```diff
@@ -0,0 +1,23 @@
+defmodule WiseHomex.DSEnergyMeter do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :ds_energy_location, WiseHomex.DSEnergyLocation
+
+    field :meter_id, :string
+    field :friendly_name, :string
+    field :meter_type, :string
+    field :comment, :string
+    field :alarm_email, :string
+    field :location_name, :string
+    field :full_location_name, :string
+    field :full_location_id, :string
```

Comment:
This is called `full_location` in the API

Reply:
Good catch :muscle: Fixed and squshed :sunny: